### PR TITLE
update branch to main

### PR DESCRIPTION
**Pull request by v1v**

### CODE REVIEW

File: .ci/jobs/defaults.yml [hunk 1]
```diff
@@ -52,7 +52,6 @@
         timeout: '15'
         use-author: true
         wipe-workspace: 'True'
-    periodic-folder-trigger: 1d
```
Comment (cachedout) [inline .ci/jobs/defaults.yml:55]: Seems unrelated to this change.

File: Jenkinsfile [hunk 2]
```diff
@@ -30,7 +30,6 @@ pipeline {
   }
   parameters {
     string(name: 'MAVEN_CONFIG', defaultValue: '-B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn', description: 'Additional maven options.')
-    booleanParam(name: 'Run_As_Master_Branch', defaultValue: false, description: 'Allow to run any steps on a PR, some steps normally only run on master branch.')
```
Comment (cachedout) [inline Jenkinsfile:33]: Seems unrelated to this change.

File: .ci/jobs/defaults.yml [hunk 3]
```diff
@@ -52,7 +52,6 @@
         timeout: '15'
         use-author: true
         wipe-workspace: 'True'
```
Comment (v1v) [inline .ci/jobs/defaults.yml:54]:
```suggestion
        wipe-workspace: 'True'
    periodic-folder-trigger: 1d
```

File: Jenkinsfile (hunk 2 above)
Comment (v1v) [reply Jenkinsfile:33]: `Master` is in there, but somehow this particular parameter is not used at all in the pipeline, so rather than changing `master` for `main` , I decided to delete the parameter itself

Comment (v1v) [reply Jenkinsfile:33]: I don't mind to revert this change if it helps

Comment (cachedout) [reply Jenkinsfile:33]: gotcha. seems fine to remove then.

**Review by cachedout — APPROVED** [review]
